web/themes/contrib/civictheme/civictheme.post_update.php

```diff
@@ -619,3 +620,97 @@ function civictheme_post_update_enable_focal_point_configurations_2(): void {
   ];
   \Drupal::classResolver(CivicthemeUpdateHelper::class)->deleteConfig($old_field_configs);
 }
+
+/**
+ * Updates blocks from 'sidebar' region to 'sidebar_top_left'.
+ *
+ * @SuppressWarnings(PHPMD.CyclomaticComplexity)
+ * @SuppressWarnings(PHPMD.StaticAccess)
+ */
+function civictheme_post_update_move_blocks_to_sidebar_top_left(): void {
+  $themes = ['civictheme'];
```

Comment:
please update to scan for **active** themes that has `civictheme` as at least one of it's `base theme` (themes can have a hierarchy of base themes).